src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -269,6 +269,8 @@ def remove_group(fs, vol_spec, groupname):
     except cephfs.Error as e:
         if e.args[0] == errno.ENOENT:
             raise VolumeException(-errno.ENOENT, "subvolume group '{0}' does not exist".format(groupname))
+        elif e.args[0] == errno.ENOTEMPTY:
```

Comment:
While this seems fine, should the check be to list the group and expect no subvolumes? @kotreshhr

Reply:
@vshankar That's an extra check. Do you think that's required ? Are there other cases were the rmdir returns ENOTEMPTY or we left with some metadata files even when subvolumes are deleted ?